packages/kestrel_core/tests/test_session.py

```diff
@@ -57,6 +57,27 @@ def test_execute_in_cache_stix_process():
             next(res)
 
 
+def test_execute_in_cache_stix_process_ocsf_disp_attr():
+    hf = """
+proclist = NEW process [ {"binary_ref.name": "cmd.exe", "pid": 123}
+                       , {"binary_ref.name": "explorer.exe", "pid": 99}
+                       , {"binary_ref.name": "firefox.exe", "pid": 201}
+                       , {"binary_ref.name": "chrome.exe", "pid": 205}
+                       ]
+DISP proclist ATTR name
+"""
+    b1 = DataFrame([ {"name": "cmd.exe"}
```

Comment:
Where is `name` supposed to come from here?

Reply:
`file.name` should work; `name` will not.  However, there is no attr mapping for `Construct` instructions at the moment, so even `file.name` won't work.  The cache has `binary_ref.name` as the column name.